tests/rgw/sanity_rgw_multisite.py

```diff
@@ -75,11 +74,27 @@ def run(**kw):
             user_details_file, test_site_node, copy_user_to_site_node, user_details_file
         )
 
-    for site in verify_io_on_sites:
-        # todo: need to implement verify io section
-        log.info(f"verification IO on {site}")
-        raise NotImplementedError("verify IO still pending")
-
+    verify_io_on_site = clusters.get(config.get("verify-io-on-site"))
+    if verify_io_on_site:
+        log.info(f'verification of IO on site: {config.get("verify-io-on-site")}')
+        verify_io_on_site_node = verify_io_on_site.get_ceph_object("rgw").node
+        io_info = home_dir_path + "io_info.yaml"
+        copy_file_from_node_to_node(
+            io_info, test_site_node, verify_io_on_site_node, io_info)
+        if test_site == 'secondary':
+            verify_out, err = primary_rgw_node.exec_command(
+                    cmd="sudo python3 " + test_folder_path + lib_dir + "read_io_info.py",
+                    timeout=timeout,
+                )
+            log.info(verify_out.read().decode())
+            log.error(verify_out.read().decode())
+        else:
+            verify_out, err = secondary_rgw_node.exec_command(
+                cmd="sudo python3 " + test_folder_path + lib_dir + "read_io_info.py",
```

Comment:
As the `read_cmd` string is being used in the else part, we could define the variable before the `if` statement to enable its usage below.